tests/io_test.py

```diff
@@ -1199,6 +1205,19 @@ def test_overwrite_single_dset(self):
 
             self.assertEqual(f1_size, f2_size)
 
+    def test_segarray_str_hdf5(self):
+        language = ak.array(["english", "spanish"])
```

Comment:
it seems we don't really use `language` anywhere

Reply:
Yeah that was from another test I did. Removed.